dags/inference/jetstream_pytorch_inference.py

```diff
@@ -48,7 +48,7 @@
           "checkpoint_quantized": "gs://inference-benchmarks/models/llama2-7b-chat/pytorch/llama-2-7b-chat-merged-int8-per-channel",
           "dataset": "openorca",
           "tokenizer": "tokenizer.llama2",
-          "batch_sizes": [96],
+          "batch_sizes": [8, 32, 96],
```

Comment:
Can you add 1-2 more size, perhaps `64`?

Reply:
added